initials.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+
+int main() {
```

Comment:
Nice job! My only suggestion is that you add a comment with your name and what this program does!